### PR TITLE
Update README doc links

### PR DESCRIPTION
**Pull request by 0candy**
https://github.com/strongloop-internal/scrum-loopback/issues/1142

### CODE REVIEW

**Comment by 0candy** [comment]
@crandmck PTAL

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (Amir-61) [inline README.md]:
```
## loopback-connector-mssql
```
Should be changed to:

```
# loopback-connector-mssql
```
@crandmck to confirm.

**Comment by 0candy** [comment]
Fixed, please review.

**Comment by crandmck** [comment]
Thanks!

**Pull request merged**